src/big/big_rational.cr

```diff
@@ -237,7 +263,18 @@ end
 struct Float
   include Comparable(BigRational)
 
+  # Returns a `BigRational` representing this float.
+  def to_big_r
+    BigRational.new(self)
+  end
+
   def <=>(other : BigRational)
     -(other <=> self)
   end
 end
+
+module Math
+  def frexp(value : BigRational)
+    frexp value.to_big_f
```

Comment:
Why not? it is used to normalize numbers (hashing, for example).

frexp exists for any number, so here just specialization for BigRational.